contracts/LevCompFactory.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "./Strategy.sol";
+
+contract LevCompFactory {
+    address public immutable original;
+
+    event Cloned(address indexed clone);
+    event Deployed(address indexed original);
+
+    constructor(
+        address _vault,
+        address _cToken
+    ) public {
+        Strategy _original = new Strategy(_vault, _cToken);
```

Comment:
why does this needs to wrap the Strategy API?

We are just storing the address not sure is needed.